src/tools/convert_dgv.py

```diff
@@ -0,0 +1,212 @@
+                break1,
+                break2,
+                opposing_strands=oppose,
+                untemplated_seq=untemplated_seq,
+                event_type=event_type,
+                stranded=stranded,
+                **{COLUMNS.tools: file_type, COLUMNS.tracking_id: std_row[COLUMNS.tracking_id]},
+            )
+
+            for col, value in std_row.items():
+                if col not in COLUMNS and col not in bpp.data:
+                    bpp.data[col] = value
+            if not event_type or event_type in BreakpointPair.classify(bpp):
+                result.append(bpp)
+        except (InvalidRearrangement, AssertionError):
+            pass
+
+    return result
+
+
+def convert_dgv_to_list(filepaths: str) -> List:
+    """
+    reads a file of regions. The expect input format for the file is tab-delimited and
+    the header should contain the following columns
+    - chr: the chromosome
+    - start: start of the region, 1-based inclusive
+    - end: end of the region, 1-based inclusive
+    - name: the name/label of the region
+    For example:
+        #chr    start       end         name
+        chr20   25600000    27500000    centromere
+    Args:
+        filepath: path to the input tab-delimited file
+    Returns:
+        lists of regions on the chromosome
+    """
+    col_list = ['chr', 'start', 'end', 'variantaccession']
+    result = []
+    rows = None
+    df = pd.read_csv(
+        filepaths,
+        sep='\t',
+        dtype={'chr': str, 'start': int, 'end': int, 'variantaccession': str},
+        usecols=col_list,
+    )
+    rows = df.where(df.notnull(), None).to_dict('records')
+    if rows:
+        for row in rows:
+            try:
+                std_rows = _convert_tool_row(row, "bed", False)
+            except Exception as err:
+                raise err
+            else:
+                result.extend(std_rows)
+    return result
+
+
+def convert_row(row: Dict) -> Dict:
+    """
+    Converts the fields from DGV output into MAVIS definitions of an SV
+    Files can be obtained from: http://dgv.tcag.ca/dgv/app/downloads?ref=GRCh37/hg19
+    Extracted BED Columns
+    - variantaccession: id for tracking information
+    - chrom: chromosome name
+    - start: start coordinate of locus
+    - end: end coordinate of locus
+    """
+    return {
+        COLUMNS.break1_chromosome: row['chr'],
+        COLUMNS.break2_chromosome: row['chr'],
+        COLUMNS.break1_position_start: row['start'],
+        COLUMNS.break1_position_end: row['start'],
+        COLUMNS.break2_position_start: row['end'],
+        COLUMNS.break2_position_end: row['end'],
+        COLUMNS.untemplated_seq: None,
+        COLUMNS.tracking_id: row['variantaccession'],
+    }
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Convert the dgv file format into a MAVIS file.",
+        add_help=False,
+    )
+    req_parser = parser.add_argument_group("required arguments")
```

Comment:
not required to change just worth noting that the argument group is superfluous when you only have 1 argument group. could just add to the regular parser